packages/datagateway-common/src/arrowtooltip.component.tsx

```diff
@@ -28,96 +29,85 @@ export const getTooltipText = (node: React.ReactNode): string => {
 const ArrowTooltip = (
   props: TooltipProps & {
     percentageWidth?: number;
```

Comment:
We don't use this prop any more do we? Can this be removed if so?

Reply:
@louise-davies It is currently used in breadcrumbs and it didn't work without it, not sure about after the refactor though.

Reply:
@joelvdavies Ah, I didn't notice that. My refactor doesn't affect the breadcrumbs themselves (only the code which renders the individual breadcrumbs) so I'd anticipate the same result but I'll double check